.github/workflows/build-checks.yml

```diff
@@ -81,6 +97,7 @@ jobs:
             -DMAD=ON \
             -DMODPLUG=ON \
             -DWAVPACK=ON \
+            -DCMAKE_CXX_FLAGS="-Wno-nan-infinity-disabled" \
```

Comment:
We should use our fpclassify.cpp instead.

Reply:
This is not a regression of this PR, as the check hasn't existed before. Therefore it's out of scope of this PR!

Reply:
Yes, but let's keep the warning until the issue is fixed in the other PR.

Reply:
But than all CI runs fail?

Reply:
Opened a new issue, so that we don't we forget about it: https://github.com/mixxxdj/mixxx/issues/13837

Problem solved?